mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoMethodGenerator.java

```diff
@@ -214,6 +218,36 @@ protected void maybeAddTimestamp(String timestamp, MethodSpec.Builder methodBuil
     }
   }
 
+  protected boolean nullSavingStrategyExplicitlySet(
+      ExecutableElement methodElement, String annotationCanonicalName) {
+    return !methodElement
+        .getAnnotationMirrors()
+        .stream()
+        .filter(
+            v -> {
+              // Find only for Update annotation
+              DeclaredType annotationType = v.getAnnotationType();
+              if (annotationType instanceof Type.ClassType) {
+                return ((Type.ClassType) annotationType)
+                    .tsym
+                    .getQualifiedName()
+                    .toString()
+                    .equals(annotationCanonicalName);
+              } else {
+                return false;
+              }
+            })
+        .flatMap(v -> v.getElementValues().values().stream())
+        .filter(v -> v instanceof Attribute.Enum)
```

Comment:
From `getElementValues()` javadocs:
> Only those elements with values explicitly present in the annotation are included, not those that are implicitly assuming their default values.

So we just need to iterate `getElementValues().keySet()` and find out if there is an element such that `key.getSimpleName().contentEquals("nullSavingStrategy")`.